web/portal/src/main/webapp/login/skin/Onboarding.css

```diff
@@ -0,0 +1,246 @@
+/**
```

Comment:
Onboarding.css -> onboarding.css
I think we should use color variables instead of static values, to verify with invision mockup